.travis.yml

```diff
@@ -0,0 +1,33 @@
+sudo: required
+dist: xenial
+
+language: python
+cache: pip
+before_install:
+  - sudo apt-get install -y libopenmpi-dev
+script:
+  - pytest yairl/
+matrix:
+  include:
+  - name: "Linting"
+    python: "3.7"
+    before_install: []
+    install:
+      - pip install flake8 isort
+    script:
+      - ci/code_checks.sh
+    after_success: []
+
+  - name: "3.7 Coverage Tests"
+    python: "3.7"
+    script:
+      - py.test --cov-report=xml --cov=yairl yairl/
```

Comment:
Add --expensive to also run the expensive tests. These are also important to make sure it works.